docs/service_architecture.md

```diff
@@ -161,11 +212,166 @@ keys in the HSM, but not to exfiltrate the private material.
   timestamps or snapshots.
 
 - **Key Compromise** - If the Notary signer uses a database as its backend,
-  an attacker can exfiltrate all the private material.  Note that the capabilities
+  an attacker can exfiltrate all the private material. Note that the capabilities
```

Comment:
It might be worth making it explicit about which private keys could be available in notary signer -- I have a feeling that "all the private material" makes it sound like the root key is in danger even though it never would be on the notary signer